Docs: add a notice in helm tutorial

LinzeSu opened the pull request:
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>


### Description of your changes
Add a notice in the Before Starting section. As the current app-with-chart-redis.yaml would fail because the chart version is outdated.
<!--

Briefly describe what this pull request does. We love pull requests that resolve an open KubeVela issue. If yours does, you
can uncomment the below line to indicate which issue your PR fixes, for example
"Fixes #500":

-->

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] Update `sidebar.js` if adding a new page.
- [ ] Run `yarn start` to ensure the changes has taken effect.


### Special notes for your reviewer

<!--

Be sure to direct your reviewers'
attention to anything that needs special consideration.

-->

wonderflow committed:
update and refine the docs especially for architecture

Signed-off-by: Jianbo Sun <jianbo.sjb@alibaba-inc.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

LinzeSu committed:
Docs: add a notice in helm tutorial

Docs: add a notice in helm tutorial

Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

dependabot[bot] committed:
Chore(deps): Bump tar from 6.1.0 to 6.1.13 (#1160)

Bumps [tar](https://github.com/npm/node-tar) from 6.1.0 to 6.1.13.
- [Release notes](https://github.com/npm/node-tar/releases)
- [Changelog](https://github.com/isaacs/node-tar/blob/main/CHANGELOG.md)
- [Commits](https://github.com/npm/node-tar/compare/v6.1.0...v6.1.13)

---
updated-dependencies:
- dependency-name: tar
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

wonderflow commented on docs/tutorials/helm.md:
```diff
@@ -11,6 +11,7 @@ Starting from here, you will learn to use the KubeVela Addons to install plug-in
 ## Before starting
 
 - Choose a Helm Chart you want to deploy. In the tutorial, we take [bitnami/redis](https://github.com/bitnami/charts/tree/master/bitnami/redis) as an example.
+- If you are using [bitnami/redis](https://github.com/bitnami/charts/tree/master/bitnami/redis), check if the chart version you specified is still listed [here](https://artifacthub.io/packages/helm/bitnami/redis) and available.
```
please also help updateing the redis version below. thanks

wonderflow replied:
please also update the zh verison, thanks

dependabot[bot] committed:
Chore(deps): Bump path-parse from 1.0.6 to 1.0.7 (#1163)

Bumps [path-parse](https://github.com/jbgutierrez/path-parse) from 1.0.6 to 1.0.7.
- [Release notes](https://github.com/jbgutierrez/path-parse/releases)
- [Commits](https://github.com/jbgutierrez/path-parse/commits/v1.0.7)

---
updated-dependencies:
- dependency-name: path-parse
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

dependabot[bot] committed:
Chore(deps): Bump normalize-url from 4.5.0 to 4.5.1 (#1162)

Bumps [normalize-url](https://github.com/sindresorhus/normalize-url) from 4.5.0 to 4.5.1.
- [Release notes](https://github.com/sindresorhus/normalize-url/releases)
- [Commits](https://github.com/sindresorhus/normalize-url/commits)

---
updated-dependencies:
- dependency-name: normalize-url
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

Arhell committed:
update dependencies (#1166)

Signed-off-by: Arhell <arhell333@gmail.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

dependabot[bot] committed:
Chore(deps): Bump ws from 7.4.4 to 7.5.9 (#1164)

Bumps [ws](https://github.com/websockets/ws) from 7.4.4 to 7.5.9.
- [Release notes](https://github.com/websockets/ws/releases)
- [Commits](https://github.com/websockets/ws/compare/7.4.4...7.5.9)

---
updated-dependencies:
- dependency-name: ws
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

LinzeSu committed:
Docs: update redis version in helm tutorial

Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

dependabot[bot] committed:
Chore(deps): Bump browserslist from 4.16.3 to 4.21.5 (#1167)

Bumps [browserslist](https://github.com/browserslist/browserslist) from 4.16.3 to 4.21.5.
- [Release notes](https://github.com/browserslist/browserslist/releases)
- [Changelog](https://github.com/browserslist/browserslist/blob/main/CHANGELOG.md)
- [Commits](https://github.com/browserslist/browserslist/compare/4.16.3...4.21.5)

---
updated-dependencies:
- dependency-name: browserslist
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

LinzeSu committed:
Docs: update redis version in helm zh tutorial

Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

Arhell committed:
update dependencies (#1170)

Signed-off-by: Arhell <arhell333@gmail.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

lunarwhite committed:
Docs: update outdated instructions for docgen (#1172)

Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

ieuD committed:
refacor: update cue for better readability (#1176)

Signed-off-by: Eren Cankurtaran <eren.cankurtaran@trendyol.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

Arhell committed:
update dependencies (#1174)

Signed-off-by: Arhell <arhell333@gmail.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

zhanghw0354 committed:
Docs: swap positions of introduces to vela port-forward and exec (#1179)

Signed-off-by: zhanghw0354 <zhanghaiwen_yewu@cmss.chinamobile.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

Arhell committed:
update dependencies (#1181)

Signed-off-by: Arhell <arhell333@gmail.com>
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>
